gammapy/modeling/models/spectral.py

```diff
@@ -1335,24 +1339,64 @@ def __init__(self, radiative_model, distance=1.0 * u.kpc, seed=None):
             parameter = Parameter("B", value)
             parameters.append(parameter)
 
+        # In case of a synchrotron self compton model, append B and Rpwn to the fittable parameters
+        if (
+            isinstance(self.radiative_model, naima.models.InverseCompton)
```

Comment:
One other option is, as we discussed un the dev call, would be to assume that all electron particle populations have a synchrotron, IC (and then probably bremsstrahlung as well). In which case, B, n and R become necessary parameters.
This might be more cumbersome and CPU intensive but less confusing? I don't really know.